kernel/environ.mli

```diff
@@ -343,13 +343,10 @@ val push_context_set : ?strict:bool -> ContextSet.t -> env -> env
     context set to the environment. It does not fail even if one of the
     universes is already declared. *)
 
-val push_qualities : Sorts.QVar.Set.t -> env -> env
-(** Add the qualities to the environment. Only used in higher layers. *)
-
 val push_quality_set : Sorts.QVar.Set.t -> env -> env
 (** [push_quality_set qs env] pushes the set of quality variables in
-    the environment. It does not fail even if a quality variable is
-    already declared. *)
+    the environment. It fails if a quality variable is already
+    declared. *)
```

Comment:
While you're at it you should rename `push_quality_set` into `push_qualities`...

Reply:
Isn't t push_context_set for univs?

Reply:
That one is called this way because of the type `ContextSet`, not because it takes sets.

Reply:
it's called contextset because it contains a set